config/default.yml

```
@@ -57,21 +57,25 @@ Rails/ActiveSupportAliases:
 Rails/ApplicationController:
   Description: 'Check that controllers subclass ApplicationController.'
   Enabled: true
+  SafeAutoCorrect: false
   VersionAdded: '2.4'
```

Can you mark `VersionChanged: '2.5'`?

```suggestion
  VersionAdded: '2.4'
  VersionChanged: '2.5'
```

Same for other changes.

---

Thx, updated && rebased && force-pushed a1aa7d5036e4af9986e5eea878d93bfe58d2b17d